### PR TITLE
Add new "machinectl shell" command for su(1)-like behaviour

### PR DESCRIPTION
**Pull request by poettering**
This adds a new "machinectl shell" command that we can use for creating su(1)-like privileged sessions, that are fully isolated from the original session.

### CODE REVIEW

**Comment by johannbg** [comment]
Took only 2 years almost to date for 1000200 bz.rh to be implemented. Better late then never ; ) This one will be appreciated by administrators ;)

File: man/machinectl.xml [hunk 1]
```diff
@@ -316,21 +378,66 @@
...
+
+        <para>When using the <command>shell</command> command without
+        arguments (thus invoking the executed shell or command on the
+        local host) it is similar in many ways to a <citerefentry
+        project='die-net'><refentrytitle>su</refentrytitle><manvolnum>1</manvolnum></citerefentry>
+        session, but unlike <command>su</command> completely isolates
+        the new session from the originating session, so that it
+        shares no process or session properties, and is in a clean and
+        well-defined state. It will be tracked in a new utmp, login,
+        audit and keyring session, and will not inherit an environment
```
Comment (vcaputo) [inline man/machinectl.xml:432]: s/an/any/

Comment (poettering) [inline man/machinectl.xml:432]: Fixed via #1048. THanks!

**Comment by kergalym** [comment]
For what it needed to use? For safety adjusting in-chroot systems?

**Comment by giannisalinetti** [comment]
I have a silly question: Is it going to coexist in parallel with the old su?

**Comment by poettering** [comment]
> For what it needed to use? For safety adjusting in-chroot systems?

`machinectl shell` is useful for getting shell sessions that are entirely isolated from the originating session  in local containers or the host.

> I have a silly question: Is it going to coexist in parallel with the old su?

Sure.

Please not that this is not a discussion forum, but an issue and patch tracker. Please redirect questions like these elsewhere, such as the systemd mailing list.

**Pull request merged**